crypto/bio/bss_conn.c

```diff
@@ -210,6 +206,12 @@ static int conn_state(BIO *b, BIO_CONNECT *c)
             break;
 
         case BIO_CONN_S_CONNECT_ERROR:
+            i = get_last_socket_error();
```

Comment:
please revert this change, reason that errno is likely no preseved over the callback.

Reply:
Good point, done.